src/audio/FaderPlaybin.h

```diff
@@ -33,8 +33,12 @@
 
 namespace Guayadeque {
 
-//#define guLogDebug(...)  guLogMessage(__VA_ARGS__)
+#ifndef NDEBUG
+#define guLogDebug(...)  guLogMessage(__VA_ARGS__)
+#else
 #define guLogDebug(...)
+#endif
```

Comment:
This will make too much output in debug build. It was just for debug faderplaybin, crossfading, recording, etc. Dont think its interesting to be enabled.